client/start.go

```diff
@@ -35,8 +35,12 @@ const (
 	stderrFile = "serverlog_stderr.txt"
 )
 
-func start(c string) (func(), error) {
+func start(c string, runInContainer bool) (func(), error) {
 	args := strings.Fields(c)
+	// If run in a container, -cmd is the name of the image.
+	if runInContainer {
+		args = []string{"docker", "run", "--network=host", c}
```

Comment:
Pass the function env vars here to the image (e.g. FUNCTION_SIGNATURE_TYPE et al)

Reply:
I imagine the function env vars are stored in the image and set by the launcher binary at run time.